.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,38 @@
+name: Feature Request/Enhancement
+description: Have a new idea of any feature you want to propose ? Please suggest.
+title: '[FEATURE] <description>'
+labels: ['goal: Addition']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the enhancement or Feature you want propose.
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Solution
+      description: Any Solution you like to propose?
+    validations:
+      required: true    
+  - type: textarea
+    id: screenshot
+    attributes:
+      label: Screenshots
+      description: Please add any screenshots if it is applicable
+    validations:
+      required: false
+  - type: textarea
+    id: additionalinfo
+    attributes:
+      label: Additional information
+      description: Is there any additional information you wish to add?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can Summit your Request.
+        
```

Comment:
I think, most of this is not really relevant, it'd be safe to omit these fields.

```suggestion
```